docs/api.md

```diff
@@ -593,6 +594,12 @@ If URLs are specified, only cookies for those URLs are returned.
   - `secure` <[boolean]>
 - returns: <[Promise]>
 
+#### page.disableCache(disabled)
```

Comment:
Let's name this `page.setCacheEnabled` to be aligned with `page.setJavascriptEnabled` method that we already have.

Reply:
I have made these changes 👍